workload/src/main/java/com/google/finapp/WorkloadMain.java

```diff
@@ -126,9 +120,18 @@ private static CommandLine parseArgs(String[] args) {
             .build());
 
     options.addOption(
-        Option.builder("n")
-            .longOpt("num-accounts")
-            .desc("number of accounts to create")
+        Option.builder("t")
+            .longOpt("thread-count")
+            .desc("number of threads to use in thread pool")
+            .required(true)
+            .type(Number.class)
+            .hasArg()
+            .build());
+
+    options.addOption(
+        Option.builder("c")
+            .longOpt("task-count")
+            .desc("number of tasks (RPC methods) each thread performs")
```

Comment:
Out of curiosity: since we keep starting new worker threads what's the difference between (1) using a limited number of tasks per worker and (2) using a limited number of workers that each generate infinitely many tasks?

Reply:
The original intention was for each client to mock a customer with a finite number of tasks but doing (2) would be more efficient. I can rework the code to do (2) if this is preferred or make an issue for it to be done in a future PR. Thoughts?

Reply:
I don't think you need to change it in this PR, just noting that (2) might be easier since you wouldn't need to queue anything, just run each worker in its own thread. FWIW since workers move balances between random accounts they don't really represent individual customers now.

Reply:
Done.